Formula/cilium-cli.rb

```diff
@@ -0,0 +1,25 @@
+class CiliumCli < Formula
+  desc "CLI to install, manage & troubleshoot Kubernetes clusters running Cilium"
+  homepage "https://cilium.io"
+  url "https://github.com/cilium/cilium-cli/archive/refs/tags/v0.8.6.tar.gz"
+  sha256 "7d4adfc511206af56570ba90dd523bd31f1703c8f9b36bca90e4d0863dd06ab7"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = "-s -w -X github.com/cilium/cilium-cli/internal/cli/cmd.Version=#{version}"
+    system "go", "build", *std_go_args(ldflags: ldflags), "-o", "#{bin}/cilium", "./cmd/cilium"
+
+    bash_output = Utils.safe_popen_read(bin/"cilium", "completion", "bash")
+    (bash_completion/"cilium").write bash_output
+    zsh_output = Utils.safe_popen_read(bin/"cilium", "completion", "zsh")
+    (zsh_completion/"_cilium").write zsh_output
+    fish_output = Utils.safe_popen_read(bin/"cilium", "completion", "fish")
+    (fish_completion/"cilium.fish").write fish_output
+  end
+
+  test do
+    assert_match(/cilium-cli: v#{version}/, shell_output("#{bin}/cilium version"))
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Unfortunately, all commands apart from `cilium help` and `cilium version` require a running Kubernetes cluster to work correctly. If that is not the case, they will just report an error message which I considered to be a bad test as well.

Any advice on how to proceed with this? I see some existing formulae also use `version` and/or `help` in their tests.

Reply:
From the linked document:

> If the software cannot function without credentials or requires a virtual machine, docker instance, etc. to run, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected. This is preferred over mocking a dependency.

Reply:
Thanks. I've updated the formula to run `cilium install` and `cilium hubble enable` and assert the errors.